src/main/java/io/strimzi/kafka/bridge/tracker/FullOffsetTracker.java

```diff
@@ -108,23 +108,13 @@ public void delivered(int partition, long offset) {
                 state.settlements.keySet().removeAll(offsetToRemove);
 
             } else {
-
                 // no other UNSETTLED offset, so the one just arrived is for commit
-
                 long offsetToCommit = offset;
                 state.offset = offsetToCommit;
                 state.flag = true;
                 // all offset SETTLED, clear list
                 state.settlements.clear();
             }
-
-        } else if (offset > state.firstUnsettled) {
-
-            // do nothing
-
-        } else {
-
-            // impossible ?
```

Comment:
can you suppress the warning here as well? It's better having these comments for now.